examples/spark2/src/main/scala/org/apache/carbondata/examples/CarbonSessionExample.scala

```diff
@@ -40,11 +40,18 @@ object CarbonSessionExample {
       .addProperty(CarbonCommonConstants.ENABLE_QUERY_STATISTICS, "false")
     val spark = ExampleUtils.createCarbonSession("CarbonSessionExample")
     spark.sparkContext.setLogLevel("error")
-    exampleBody(spark)
+    Seq(
+      "stored as carbondata",
+      "using carbondata",
+      "stored by 'carbondata'",
```

Comment:
Is it still supported?

Reply:
yes, CarbonSession will keep all old features.